### ISSUE TITLE
backpropagation through a sparse tensor

### ISSUE THREAD

**Issue by hua-wu-que** [StanfordVL/MinkowskiEngine#147]
Hi there,

I have a network G that generate a feature vector "feats" of size [b,n] together with a position vector "coords" of size [b, 3]
I then create a sparse tensor using 
``
        sinput = ME.SparseTensor(
            feats=feats.cpu(),
            coords=ME.utils.batched_coordinates(coords),
            quantization_mode=ME.SparseTensorQuantizationMode.UNWEIGHTED_AVERAGE
        ).to(feats)
``
Next, I feed sinput into a MinkowskiConvolution layer  defined as 
``
        self.enc_block_s1 = nn.Sequential(
            ME.MinkowskiConvolution(
                ch[0], ch[1], kernel_size=3, stride=1, dimension=3),
            ME.MinkowskiBatchNorm(ch[1]),
            ME.MinkowskiELU(),
        ).cuda()

loss = self.enc_block(feat).F.mean().backward()
``
But calling loss.backward() cannot backpropage gradient to network G, (printing G's parameters gradient got None).

could you give me some suggestions on how to do this backward?

Many thanks!
